server.js

```diff
@@ -61,9 +61,6 @@ function twoStatic(url, file) {
 }
 
 twoBrowserified("/katex.js", "katex", "katex");
-twoUse("/test/jasmine", express.static(path.dirname(
-    require.resolve("jasmine-core/lib/jasmine-core/jasmine.js"))));
-twoBrowserified("/test/katex-spec.js", "test/*[Ss]pec.js");
```

Comment:
Could we replace this with something like [jest-html-reporter](https://www.npmjs.com/package/jest-html-reporter) (as a dev dependency)?  Or do we not care enough about this functionality?